packages/client/src/runtime/core/engines/binary/BinaryEngine.ts

```diff
@@ -194,11 +191,6 @@ export class BinaryEngine extends Engine<undefined> {
     this.previewFeatures = this.previewFeatures.filter((e) => !removedFlags.includes(e))
     this.engineEndpoint = engineEndpoint
 
-    if (engineEndpoint) {
```

Comment:
Isn't this an internal thing that allowed to connect client to a running engine? Or engine people do not need this anymore?

Reply:
It's still there and should be working, but assigning `this.port` is not really necessary here because the code path where `this.port` was used is completely bypassed when `engineEndpoint` is specified, so this `if` statement essentially was just dead code.